build.gradle

```diff
@@ -243,15 +240,19 @@ dependencies {
   functionalTestCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: springBootVersion
 }
 
-jar {
-  archiveName 'send-letter-service.jar'
+mainClassName = 'uk.gov.hmcts.reform.sendletter.Application'
+
+bootJar {
+  archiveName = 'send-letter-service.jar'
 
   manifest {
     attributes('Implementation-Version': project.version.toString())
   }
 }
 
-task migratePostgresDatabase(type: org.flywaydb.gradle.task.FlywayMigrateTask) {
+import org.flywaydb.gradle.task.FlywayMigrateTask
```

Comment:
do we need to do this ?

Reply:
as part of upgrade: solve soft warnings in gradle scripts